core/src/main/resources/application.conf

```diff
@@ -118,6 +120,11 @@ call-caching {
   // request to an external service (DockerHub, GCR).  If a call fails to lookup a
   // Docker hash, it will fail.
   lookup-docker-hash = false
+
+  // This option determines whether file contents are hashed to determine cache hit or miss.
+  hash-file-contents = true
+  // This option determines whether file paths are hashed to determine cache hit or miss.
+  hash-file-paths = false
```

Comment:
I could see myself wanting that behavior. Im telling it I don't care about the real docker hash, so I shouldn't care